### ISSUE TITLE
NameError: name 'basestring' is not defined

### ISSUE THREAD

**Issue by p0f1g1st** [aboul3la/Sublist3r#120]
Hello! When I search by some engines (usually Google) I get this error:
Process GoogleEnum-2:
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/home/p0f1g1st/tools/Sublist3r/sublist3r.py", line 264, in run
    domain_list = self.enumerate()
  File "/home/p0f1g1st/tools/Sublist3r/sublist3r.py", line 235, in enumerate
    if not self.check_response_errors(resp):
  File "/home/p0f1g1st/tools/Sublist3r/sublist3r.py", line 299, in check_response_errors
    if isinstance(resp, basestring) and 'Our systems have detected unusual traffic' in resp:
NameError: name 'basestring' is not defined

**Comment by jungle-boogie** [comment]
Me too!

```
Process GoogleEnum-4:
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "sublist3r.py", line 264, in run
    domain_list = self.enumerate()
  File "sublist3r.py", line 235, in enumerate
    if not self.check_response_errors(resp):
  File "sublist3r.py", line 299, in check_response_errors
    if isinstance(resp, basestring) and 'Our systems have detected unusual traffic' in resp:
NameError: name 'basestring' is not defined
```

```
$ python
Python 3.6.3 (default, Nov 15 2017, 14:42:38) 
[GCC 4.2.1 Compatible OpenBSD Clang 5.0.0 (tags/RELEASE_500/final)] on openbsd6
Type "help", "copyright", "credits" or "license" for more information.

**Comment by shiran3F** [comment]
basestring is no longer available in Python 3. [From What’s New In Python 3.0:](https://docs.python.org/3.0/whatsnew/3.0.html#text-vs-data-instead-of-unicode-vs-8-bit)

> The builtin **_basestring_** abstract type was removed. Use _**str**_ instead. The **_str_** and **_bytes_** types don’t have functionality enough in common to warrant a shared base class. The **_2to3_** tool (see below) replaces every occurrence of **_basestring_** with **_str_**.

So just replace basestring with str.

**Comment by aboul3la** [comment]
Sorry for this issue. I just pushed a fix for it in commit 832d544fa643928f7fd5a5a556a4bbe5624bbce0  .

Please update to the latest version using command ```git pull```1
